socorro/scripts/fetch_crash_data.py

```diff
@@ -82,9 +83,8 @@ def fetch_crash(
 
         # Save raw crash to file system
         raw_crash = resp.json()
-        fn = os.path.join(
-            outputdir, "v2", "raw_crash", crash_id[0:3], "20" + crash_id[-6:], crash_id
-        )
+        key = build_keys("raw_crash", crash_id)[0]
```

Comment:
This script uses the RawCrash API to download the data and uses `build_keys` to figure out where on the file system to put the file. Because of that, we only need to look at the first item in the keys list.